src/containers/PageBuilder/BlockBuilder/BlockSocialMediaLink/BlockSocialMediaLink.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { func, node, object, shape, string } from 'prop-types';
+import classNames from 'classnames';
+
+import Field, { hasDataInFields } from '../../Field';
+import BlockContainer from '../BlockContainer';
+
+import css from './BlockSocialMediaLink.module.css';
+
+const BlockSocialMediaLink = props => {
+  const { blockId, className, rootClassName, link } = props;
+
+  const classes = classNames(rootClassName || css.root, className);
+
+  return (
+    <BlockContainer id={blockId} className={classes}>
+      <Field data={link} />
+    </BlockContainer>
+  );
+};
+
+const propTypeOption = shape({
+  fieldComponents: shape({ component: node, pickValidProps: func }),
+});
+
+BlockSocialMediaLink.defaultProps = {
+  className: null,
+  rootClassName: null,
+  mediaClassName: null,
+  textClassName: null,
+  ctaButtonClass: null,
+  title: null,
+  text: null,
+  callToAction: null,
+  media: null,
+  responsiveImageSizes: null,
+  options: null,
+};
+
+BlockSocialMediaLink.propTypes = {
+  blockId: string.isRequired,
+  className: string,
+  rootClassName: string,
+  mediaClassName: string,
+  textClassName: string,
+  ctaButtonClass: string,
+  title: object,
+  text: object,
+  callToAction: object,
+  media: object,
+  responsiveImageSizes: string,
+  options: propTypeOption,
+};
```

Comment:
You only extract these props: blockId, className, rootClassName, link
Therefore, I guess the other propTypes could be removed.

However, you should pass the **_options_** concept to `<Field>`. That makes this whole PageBuilder setup extendable.